modules/update-service-overview.adoc

```diff
@@ -1,28 +1,35 @@
 // Module included in the following assemblies:
 //
 // * architecture/architecture.adoc
+// * upgrading/upgrading-aws.adoc
 
 [id='update-service-overview-{context}']
-== The {product-title} update service
+== About the {product-title} update service
 
-The {product-title} update service is the hosted service that provides over the air updates to both
-{product-title} and {op-system-first}. It provides a graph, or diagram that contain
-_vertices_ and the _edges_ that connect them, of component Operators. The edges
-in the graph show which versions you can safely upgrade to, and the vertices
-are update payloads that specify the intended state of the managed cluster components.
-The Cluster Version Operator checks with the {product-title} update service and determines valid upgrades and upgrade paths
-based on current component versions and information in the graph. If you
-configure it to do so, the {product-title} update service sends the release artifacts that it needs to
-perform the upgrade to your image registry, and the Cluster Version Operator
-upgrades your cluster. By accepting automatic updates, you can automatically
+The {product-title} update service is the hosted service that provides over the
+air updates to both {product-title} and {op-system-first}. It provides a graph,
+or diagram that contain _vertices_ and the _edges_ that connect them, of
+component Operators. The edges in the graph show which versions you can safely
+upgrade to, and the vertices are update payloads that specify the intended state
+of the managed cluster components.
+
+The Cluster Version Operator (CVO) checks with the {product-title} update
+service and determines valid upgrades and upgrade paths based on current
+component versions and information in the graph. When you request an upgrade,
```

Comment:
When an upgrade is requested, CVO just uses the release image as specified by the Update / the graph and upgrades the cluster to the release image. It does not push it to any image registry.